### PR TITLE
metadata version bump

### PR DESCRIPTION
**Pull request by MilutinSekulic**
Changes:
- Metadata version bump.

### CODE REVIEW

**Review by eduardogranawd — APPROVED** [review]
LGTM

File: wd_blender_tools/addon_static.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 """Static data for the add-on except for text messages. Those are in lib/text_static"""
 
-blender_addon_version = (1, 0, 0)
+blender_addon_version = (1, 0, 1)
```
Comment (eduardogranawd) [inline wd_blender_tools/addon_static.py:8]: Hey @MilutinSekulic  is this metadata or addon version?

Comment (MilutinSekulic) [reply wd_blender_tools/addon_static.py:8]: That one is metadata. Although we were, at the time, treating both the same for simplicity. As agreed we will be separating it.

**Review by eduardogranawd — APPROVED** [review]
LGTM

**Pull request merged**